nrf5340-net-hal/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "nrf5340-net-hal"
+version = "0.1.0"
```

All crate versions are kept in sync, since that simplifies maintenance and usage

```suggestion
version = "0.15.0"
```

This should make CI get a bit further